Gems/ROS2/Code/Include/ROS2/Manipulator/ManipulatorControllerComponent.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TickBus.h>
+#include <AzCore/Name/Name.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+#include <ROS2/VehicleDynamics/DriveModels/PidConfiguration.h>
+
+#include <control_msgs/action/follow_joint_trajectory.hpp>
+#include "rclcpp_action/server.hpp"
+#include "rclcpp_action/rclcpp_action.hpp"
+
+
+namespace ROS2
+{
+    enum class GoalStatus
+    {
+        Pending,
+        Active,
+        Concluded
+    };
+
+    class FollowJointTrajectoryActionServer
+    {
+    public:
+        using GoalHandleFollowJointTrajectory = rclcpp_action::ServerGoalHandle<control_msgs::action::FollowJointTrajectory>;
+        using FollowJointTrajectory = control_msgs::action::FollowJointTrajectory;
+        FollowJointTrajectoryActionServer();
+        // std::shared_ptr<GoalHandleFollowJointTrajectory> GetGoal();
```

Comment:
unused variables should be removed  before merging, even if commented out

Reply:
Thanks, it was an oversight